<launch>

  
  <node pkg="nodelet" type="nodelet" name="multi_robot_patrol_manager" args="manager" output="screen" >
    <rosparam>
        num_worker_threads: 32
    </rosparam>
  </node>
  
  <node pkg="nodelet" type="nodelet" name="robot_follow" args="load micros_mars_task_alloc/RobotFollow multi_robot_patrol_manager">
  </node>

</launch>
